setup.py

```diff
@@ -75,6 +75,7 @@
         'requests',
         'networkx>=2.0.0',
         'rpcq>=2.7.2',
+        'ipython', # for pyquil.latex.ipython
```

Comment:
can we make this optional somehow? I'm happy including it in the examples requirements but the pyquil library doesnt need ipython when not being used in an interactive fashion